content/index.fr.md

```diff
@@ -0,0 +1,47 @@
+
+[[Éléments essentiels de l'accessibilité Web]](/fundamentals/components/)
+:   Montre comment l'accessibilité Web dépend de l'association entre les nombreuses composantes du développement Web et de l'interaction utilisateur, et le lien entre les différentes règles de la WAI : les Règles pour l'accessibilité des contenus Web ([WCAG](/standards-guidelines/wcag/), les Règles d'accessibilité pour les outils d'édition ([ATAG](/standards-guidelines/atag/) et les Règles pour l’accessibilité des agents utilisateurs ([UAAG](/standards-guidelines/uaag/).  
+
+[[Principes d'accessibilité]](/fundamentals/accessibility-principles/)
+:   Aide les développeurs, les designers, et tout autre intervenant à comprendre les principes pour créer des sites et des applications Web, des navigateurs, et d'autres outils accessibles. Fournit des liens vers les standards internationaux de l'Initiative pour l'accessibilité du Web (WAI) du W3C et vers une page dédiée aux [témoignages d'utilisateurs du Web](/people-use-web/user-stories/).
+
+[[L'accessibilité Web illustrée : explorer l'impact et les bénéfices pour tous]](/perspective-videos/)
+:   **Vidéos** démontrant le caractère essentiel de l'accessibilité Web pour les personnes en situation de handicap et son utilité pour tous. Les vidéos suivantes exemplifient l'impact de l'accessibilité et les bénéfices pour tous dans diverses situations. Chaque vidéo dure environ 1 minute et la [compilation](https://www.youtube.com/watch?v=3f31oufqFSM) dure 7 min 37 s. Chaque page contient des informations supplémentaires.
+
+[[Comment les personnes handicapées utilisent le Web]](/people-use-web/)
+:   Introduit comment les personnes porteuses de handicap utilisent le Web. Décrit les outils et les solutions pour lesquelles les personnes qui présentent différents types de handicaps optent pour naviguer sur le Web et les obstacles auxquels elles se heurtent lorsque le design n'est pas optimal.
+
+[[Utilisateurs plus âgés et accessibilité Web : répondre aux besoins des utilisateurs Web plus âgés]](/older-users/)
+:   Explique que la création de produits plus faciles à utiliser pour les personnes âgées est similaire à la création des produits pour les personnes handicapées. Les règles à appliquer pour créer des sites, des applications et des outils Web visant à faciliter l'expérience des personnes âgées sont reprises dans les standards internationaux actuels d'accessibilité  du W3C, dont les Règles pour l'accessibilité des contenus Web (WCAG).
+    -   [Développer des sites Web pour les personnes âgées : comment les Règles pour l'accessibilité des contenus Web (WCAG) 2.0 s'appliquent](https://www.w3.org/WAI/older-users/developing/)
+    -   [L'accessibilité Web pour les utilisateurs plus âgés : un aperçu de la littérature](https://www.w3.org/WAI/older-users/literature/)
+
+[[Accessibilité, ergonomie, et inclusion]](/fundamentals/accessibility-usability-inclusion/)
```

Comment:
English: "Accessibility, Usability, and Inclusion"

French translation: Accessibilité, ergonomie, et inclusion (en anglais)

I was just wondering if "usabilité" could be used or is "ergonomie" the usual translation for usability, usability testing etc.

Reply:
Reply from @remibetin
That is a tricky one, also a good candidate for glossary discussion.

Discussing with French usability professionals, I found out that "test d'utilisabilité" or "test utilisateur" were the most used for "usability testing". See my remark here : Soahmed2031/wai-InvolveUsersEval#1 (comment)
But in some other contexts, "ergonomie" may be the right word. For example, I would translate "usability professionals" into "ergonomes".

Reply:
@remibetin - I was rethinking this one:  in fact, "test d'utilisabilité" wouldn't be really exact as we don't speak about usability testing. What about "accessibilité, utilisabilité et inclusion (en anglais)", i.e. without the word "test".  Would that work?

Reply:
@vmmiller Yes of course, I cited "test d'utilisabilité" because I came across "usability" in this context.